File: 3536-maximum-product-of-two-digits/3536-maximum-product-of-two-digits.c
int maxProduct(int n) {
    int max = 0;
    int second = 0;
    while(n != 0){
        int temp = n % 10;
        if(temp >= max){
            second = max;
            max = temp;
        }else if(temp > second){
            second = temp;
        }
        n = n / 10;
    }
    return max*second;
}

File: 3536-maximum-product-of-two-digits/3536-maximum-product-of-two-digits.py
class Solution:
    def maxProduct(self, n: int) -> int:
        maxx = 0
        second = 0
        while n != 0:
            temp = n % 10
            if maxx <= temp:
                second = maxx
                maxx = temp
            elif second < temp:
                second = temp
            n = n // 10
        return maxx*second